python/GafferUI/SpreadsheetUI/_Metadata.py

```diff
@@ -181,6 +181,32 @@
 
 		],
 
+		"resolvedRows" : [
+
+			"description",
+			"""
+			An output plug containing the resolved cell values for all active
+			rows, This can be used to drive expressions in situations where the
+			standard `out` plug is not useful, or would be awkward to use. The
+			values are formatted as follows :
+
+			```
+			[
+			    { "name" : "row1Name", "cells" : { "columnName", columnValue, ... } },
+			    { "name" : "row2Name", "cells" : { "columnName", columnValue, ... } },
```

Comment:
This is supposed to be `"columeName" : ` not `"columeName", `, right?  Indicating nested dictionaries?

Reply:
Yep - I've just squashed in a fix.